src/base/static/js/views/app-view.js

```diff
@@ -350,46 +332,36 @@ export default Backbone.View.extend({
         store.dispatch(setLayerStatus(layerId, layerStatus)),
     });
 
-    if (this.options.rightSidebarConfig.is_enabled) {
-      $("body").addClass("right-sidebar-active");
-      if (this.options.rightSidebarConfig.is_visible_default) {
-        $("body").addClass("right-sidebar-visible");
-      }
-
-      // REACT PORT SECTION ///////////////////////////////////////////////////
-      ReactDOM.render(
-        <Provider store={store}>
-          <ThemeProvider theme={theme}>
-            <ThemeProvider theme={adjustedTheme}>
-              <RightSidebar
-                placeCollectionsPromise={placeCollectionsPromise}
-                places={this.places}
-                router={this.options.router}
-              />
-            </ThemeProvider>
-          </ThemeProvider>
-        </Provider>,
-        document.getElementById("right-sidebar-container"),
-      );
-      // END REACT PORT SECTION ///////////////////////////////////////////////
-    }
+    // Load activities from the API
+    _.each(this.activities, function(collection, key) {
+      collection.fetch({
+        reset: true,
+        attribute: "target",
+        attributesToAdd: {
+          datasetId: _.find(self.options.mapConfig.layers, function(layer) {
+            return layer.id == key;
+          }).id,
+          datasetSlug: _.find(self.options.mapConfig.layers, function(layer) {
+            return layer.id == key;
+          }).slug,
+        },
+      });
+    });
 
-    // REACT PORT SECTION /////////////////////////////////////////////////////
-    if (this.options.leftSidebarConfig.is_enabled) {
-      ReactDOM.render(
-        <Provider store={store}>
-          <LeftSidebar />
-        </Provider>,
-        document.getElementById("left-sidebar-container"),
-      );
-    }
-    // END REACT PORT SECTION /////////////////////////////////////////////////
+    // TODO(luke): When AppView is ported to a react component, await
+    // the place collections to load in the App component:
+    placeCollectionsPromise.then(fetchedCollections => {
+      const allPlaces = fetchedCollections.reduce((places, collection) => {
+        return [...collection.models.map(model => model.toJSON()), ...places];
+      }, []);
+      store.dispatch(setPlaces(allPlaces));
```

Comment:
Is this the only point where the `places` duck gets populated with models? If so I think we need some way to update it when a user submits a new Place. Right now if I submit a new Place to the map then jump to the ListView, the new place does not appear there.

Reply:
Perhaps we want to maintain a Backbone collection listener for the time being for this purpose.